pkg/monitor/datapath_trace_test.go

```diff
@@ -123,40 +122,32 @@ func (s *MonitorSuite) TestDecodeTraceNotifyErrors(c *C) {
 	c.Assert(err.Error(), Equals, "Unrecognized trace event (version 255)")
 }
 
-func BenchmarkDecodeTraceNotifyVersion0(b *testing.B) {
+func (s *MonitorSuite) BenchmarkDecodeTraceNotifyVersion0(c *C) {
 	input := TraceNotifyV0{}
 	buf := bytes.NewBuffer(nil)
 
-	if err := binary.Write(buf, byteorder.Native, input); err != nil {
-		b.Fatal(err)
-	}
-
-	b.ReportAllocs()
```

Comment:
is the call `b.ReportAllocs()` needs to be replaced by `c.ReportAllocs()` or it is not supported?

Reply:
I wasn't able to run the gocheck benchmark though, https://stackoverflow.com/questions/24421535/how-do-you-use-the-benchmark-flags-for-the-go-golang-gocheck-testing-framework is the only thing that I found but wasn't helpful. I'm tempted to drop the last commit.

Reply:
Dropped the last commit.